cmd/stack/deploy.go

```diff
@@ -73,7 +73,16 @@ func Deploy(ctx context.Context) *cobra.Command {
 			}
 
 			err = stack.Deploy(ctx, s, options)
-			analytics.TrackDeployStack(err == nil, s.IsCompose)
+
+			cwd, err := os.Getwd()
+			if err != nil {
+				oktetoLog.Infof("failed to get the current working directory in up: %v", err)
+			}
+			repo, err := model.GetRepositoryURL(cwd)
+			if err != nil {
+				oktetoLog.Info("could not get repository from current dir")
+			}
+			analytics.TrackDeployStack(err == nil, s.IsCompose, repo)
```

Comment:
we should lo this for `okteto deploy`, and check there if it's a compose or not. `stack` commands will be deprecated, no?

Reply:
Stack will be deprecated but it will still be a hidden command to deploy. So until we remove them and refactor stack to be a command not callable I think it's good to have it. WDYT

Reply:
agree, but we need to implement this on `okteto deploy` too

Reply:
In that case, do we have to send an event for every command on the deploy section? `okteto deploy` can deploy more than one type per execution

Reply:
for simplicy, I would go with each subcommand sending its own metrics